### PR TITLE
Voting System (again)

### PR DESCRIPTION
**Pull request by adacovsk**
[tested]

Added two lines to make this work properly. It's #31884 but fixed

 * rscadd: Admins have more options in the vote verb

### CODE REVIEW

**Comment by Kromkar** [comment]
Every time you code something it's either broken or it breaks the server

**Comment by adacovsk** [comment]
> Every time you code something it's either broken or it breaks the server

Since when? This is the only time I've done anything that broke. Downvoting something that improves things is stupid and you have no idea what you're talking about.

**Comment by Armadingus** [comment]
Can you explain what this PR does to someone who isn't fluent in byond? What options do admins get?

**Comment by adacovsk** [comment]
> Can you explain what this PR does to someone who isn't fluent in byond? What options do admins get?

you can call a map vote, you can switch from weighted to majority voting, and switch from compiled maps to votable maps with the vote verb

**Comment by Lumbermancer** [comment]
Doesn't seem to have any downsides.

File: code/modules/html_interface/voting/voting.dm [hunk 1]
```diff
@@ -19,11 +19,11 @@ var/global/datum/controller/vote/vote = new()
 /datum/html_interface/nanotrasen/vote/Topic(href, href_list[])
 	..()
 	if(href_list["html_interface_action"] == "onclose")
-
-		var/datum/html_interface_client/hclient = getClient(usr.client)
+		var/mob/user = usr
```
Comment (DamianX) [inline code/modules/html_interface/voting/voting.dm]: What does this do? `usr` is already a var of type `/mob`

File: code/controllers/configuration.dm [hunk 2]
```diff
@@ -32,6 +32,8 @@
 	var/allow_admin_ooccolor = 0		// Allows admins with relevant permissions to have their own ooc colour
 	var/allow_vote_restart = 0 			// allow votes to restart
 	var/allow_vote_mode = 0				// allow votes to change mode
+	var/toggle_maps = 0					// Change from votable maps = 0 to all compiled maps = 1
+	var/toggle_vote_method = 0			// Toggle voting methods: Weighted = 0, Majority = 1
```
Comment (DamianX) [inline code/controllers/configuration.dm:36]: Looks like these variables are only ever read from, never set

**Review by DamianX — COMMENTED** [review]
I have no idea what any of this does

File: code/controllers/configuration.dm (hunk 2 above)
Comment (adacovsk) [reply code/controllers/configuration.dm:36]: These are default values and are set in the voting.js interface

Comment (DamianX) [reply code/controllers/configuration.dm:36]: That doesn't seem right. Elaborate on that?

**Pull request merged**